### ISSUE TITLE
Webpack v5: Plugin is not cleaning up chunk/module correctly

### ISSUE THREAD

**Issue by cascornelissen** [cascornelissen/svg-spritemap-webpack-plugin#127]
@cascornelissen 

**v3.6.0** create `spritemap.js` file.
Do you have option to delete this file?

Related? #17

_Originally posted by @sectsect in https://github.com/cascornelissen/svg-spritemap-webpack-plugin/issues/108#issuecomment-717617447_

**Comment by cascornelissen** [comment]
> @sectsect, thanks for reporting the issue, it should be deleted automatically (unless `chunk.keep` is set to `true`), there's even a test for it which seems to be green for both Webpack 4 and 5 so I'll investigate manually.

**Comment by sectsect** [comment]
@cascornelissen 

I created a minimal project.
https://github.com/sectsect/svg-spritemap-webpack-plugin-test

1. `npm run build`
2. Check directory `dist/assets/js`
    ```
    $ ls dist/assets/js
    hello.js        spritemap.js
    $ cat dist/assets/js/spritemap.js
    (self.webpackChunkhello=self.webpackChunkhello||[]).push([[355],{256:function(){}}]);SECT-iMac:svg-spritemap-webpack-plugin-test
    ```

**Comment by cascornelissen** [comment]
This should be fixed in `3.6.1` which is now available under the `next` dist-tag on NPM.

```
$ npm install svg-spritemap-webpack-plugin@next
```

Please test this release out using both Webpack 4 and 5 if possible and report any problems.
